networks/suzuka/suzuka-config/src/lib.rs

```diff
@@ -1,36 +1,44 @@
+use thiserror::Error;
+
+#[derive(Error, Debug)]
+pub enum ConfigError {
+	#[error("Could not read the Environment variables")]
+	EnvReadError,
+	#[error("Could not write the Environment variables")]
+	EnvWriteError,
+	#[error("Could not write the Bash export string")]
+	WriteBashExportStringError,
+}
+
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Config {
-    pub execution_config : maptos_execution_util::config::Config,
+	pub execution_config: maptos_execution_util::config::Config,
 }
 
 impl Config {
+	pub fn new(execution_config: maptos_execution_util::config::Config) -> Self {
+		Self { execution_config }
+	}
+
+	pub fn try_from_env() -> Result<Self, ConfigError> {
+		let execution_config = maptos_execution_util::config::Config::try_from_env()
+			.map_err(|_| ConfigError::EnvReadError)?;
```

Comment:
If you had `maptos_execution_util::config::Config::try_from_env()` return `Result<Self, ConfigError>`, as opposed to `Result<Self, anyhow::Error>`, I don't think you would have to call `map.err` and could just `?` directly on `try_from_env`